src/utils/updater/updater.go

```diff
@@ -0,0 +1,43 @@
+package updater
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/Arriven/db1000n/src/runner/config"
+)
+
+func Run(configPaths []string, backupConfig []byte) {
+	lastKnownConfig := &config.RawConfig{Body: backupConfig}
+
+	for {
+		rawConfig := config.FetchRawConfig(configPaths, lastKnownConfig)
+
+		if !bytes.Equal(lastKnownConfig.Body, rawConfig.Body) {
+			file, err := os.Create("config/config.json")
```

Comment:
Can we expose this path via commandline/env variable? Can be done in a separate PR